### DISCUSSION TITLE
Copilot Commit Message Instructions

### DISCUSSION THREAD

**Discussion by yahoai** [community/community#94917] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Having predefined commit message instructions for organizations or individuals can significantly enhance workflow efficiency.

By implementing a system where commit messages automatically align with established commit conventions, the process is streamlined, ensuring consistency and clarity in version control.

This approach not only saves time but also enables the maintenance of high-level documentation, which is essential for effective team collaboration and project management

**Comment by nalbion** [comment]
This a good idea, some people I work with prefix every commit with an emoji, I've also seen other conventions like `[fix]`, `[lint]` and task/story ID etc.

**Comment by Tanimou** [comment]
Also the ability to custom the promp for commit message might be very helpful
